### PR TITLE
PGD reedit group 16

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

File: product_docs/docs/pgd/5/postgres-configuration.mdx [hunk 1]
```diff
@@ -36,7 +36,7 @@ which vary according to the size and scale of the cluster:
     -  One per database on that instance
     -  Four per PGD-enabled database
     -  One per peer node in the PGD group 
-    -  One for each writer-enabled per peer node in the PGD group
+    -  One for each writer-enabled per peer node in the PGD group
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/postgres-configuration.mdx:39]: Should the word "per" in this item be deleted?

File: product_docs/docs/pgd/5/postgres-configuration.mdx [hunk 2]
```diff
@@ -51,10 +51,10 @@ N slots and WAL senders. During synchronization, PGD temporarily uses another
 N-1 slots and WAL senders, so be careful to set the parameters high enough
 for this occasional peak demand.
 
-With parallel apply turned on, the number of slots must be increased to
-N slots from the formula \* writers. This is because the `max_replication_slots`
+With Parallel Apply turned on, the number of slots must be increased to
+N slots from the formula \* writers. This is because `max_replication_slots`
 also sets the maximum number of replication origins, and some of the functionality
-of parallel apply uses extra origin per writer.
+of Parallel Apply uses extra origin per writer.
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/postgres-configuration.mdx:57]: Is this missing the word "an," i.e., "an extra origin per writer"?

File: product_docs/docs/pgd/5/repsets.mdx [hunk 3]
```diff
@@ -13,7 +13,7 @@ nodes.
 Every PGD group creates a replication set with the same name as
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/repsets.mdx:13]: In the above, is a noun missing after "multi-master?

File: product_docs/docs/pgd/5/repsets.mdx [hunk 4]
```diff
@@ -249,29 +249,29 @@ You can use [`bdr.replication_set_add_ddl_filter`](/pgd/latest/reference/repsets
...
 
-There's also, as we recommend, a witness node, named `witness` in `region-c`, but we won't be needing to mention that in this example. The cluster itself will be called `sere`.
+There's also, as we recommend, a witness node named `witness` in `region-c` that isn't mentioned in this example. The cluster is called `sere`.
 
 This configuration looks like this:
 
 ![Multi-Region 3 Nodes Configuration](./planning/images/always-on-2x3-aa-updated.png)
 
-This is the standard Always-on multiregion configuration as discussed in the [Choosing your architecture](planning/architectures) section.
+This is the standard Always-on Multi-location configuration discussed in [Choosing your architecture](planning/architectures).
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/repsets.mdx:266]: I looked in the architecture doc and only saw multi-location, not multi-region. Please verify.

File: product_docs/docs/pgd/5/sequences.mdx [hunk 5]
```diff
@@ -181,15 +181,15 @@ The differences between timeshard and SnowflakeId are as following:
...
 currently assigned local range is used up.
 
 Unlike SnowflakeId sequences, `galloc` sequences support all sequence data types
 provided by PostgreSQL: `smallint`, `integer`, and `bigint`. This means that
 you can use `galloc` sequences in environments where 64-bit sequences are
-problematic. Examples include using integers in javascript, since that supports only
+problematic. Examples include using integers in JavaScript, since that supports only
 53-bit values, or when the sequence is displayed on output with limited space.
 
 The range assigned by each voting is currently predetermined based on the
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/sequences.mdx:195]: please verify the above sentence is correct. It looks like maybe a noun is missing either after "each" or after "voting"?

File: product_docs/docs/pgd/5/postgres-configuration.mdx (hunk 1 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/postgres-configuration.mdx:39]: I don't think so. It's not right though....
It's the number of peer nodes times the (number of writers (bdr.num_writers) plus one).

There's a spreadsheet that has the calculations which has to date defied being written as text.

File: product_docs/docs/pgd/5/postgres-configuration.mdx (hunk 2 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/postgres-configuration.mdx:57]: yes, an extra,

File: product_docs/docs/pgd/5/repsets.mdx (hunk 3 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/repsets.mdx:13]: topologies

File: product_docs/docs/pgd/5/repsets.mdx (hunk 4 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/repsets.mdx:266]: Same thing. This is from when we used location and region interchangably. Use multi-region as the architecture docs will be being redone.

File: product_docs/docs/pgd/5/repsets.mdx [hunk 6]
```diff
@@ -334,20 +334,20 @@ SELECT relname, set_name FROM bdr.tables ORDER BY relname, set_name;
 (3 rows)
 ```
 
-We want the `opinion` table to be replicated only within `region_a`, and separately only within `region_b`. To do that, we add the table to the replica sets of each region.
+You want the `opinion` table to be replicated only in `region_a` and, separately, only in `region_b`. To do that, you add the table to the replica sets of each region:
```
[the remark this answers is not in the document]
Comment (djw-m) [reply product_docs/docs/pgd/5/repsets.mdx:337]: Yes, only region a will replicate changes in region a, not send it over to region b, and vice versa. This isolates the two regions.

File: product_docs/docs/pgd/5/sequences.mdx (hunk 5 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/sequences.mdx:195]: each voting node

I believe.

File: product_docs/docs/pgd/5/postgres-configuration.mdx (hunk 1 above)
Comment (ebgitelman) [inline product_docs/docs/pgd/5/postgres-configuration.mdx:39]:
```suggestion
    -  The number of peer nodes times the (number of writers (bdr.num_writers) plus one)
```

File: product_docs/docs/pgd/5/postgres-configuration.mdx (hunk 2 above)
Comment (ebgitelman) [inline product_docs/docs/pgd/5/postgres-configuration.mdx:57]:
```suggestion
of Parallel Apply uses an extra origin per writer.
```

File: product_docs/docs/pgd/5/repsets.mdx (hunk 4 above)
Comment (ebgitelman) [inline product_docs/docs/pgd/5/repsets.mdx:266]:
```suggestion
This is the standard Always-on Multi-region configuration discussed in [Choosing your architecture](planning/architectures).
```

File: product_docs/docs/pgd/5/sequences.mdx (hunk 5 above)
Comment (ebgitelman) [inline product_docs/docs/pgd/5/sequences.mdx:195]:
```suggestion
The range assigned by each voting node is currently predetermined based on the
```

**Review by djw-m — APPROVED** [review]
LGTM

**Pull request merged**